tests/test_models/test_backbones/utils.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from torch.nn.modules import GroupNorm
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from mmdet.models.backbones.res2net import Bottle2neck
+from mmdet.models.backbones.resnet import BasicBlock, Bottleneck
+from mmdet.models.backbones.resnext import Bottleneck as BottleneckX
+from mmdet.models.utils import SimplifiedBasicBlock
+
+
+def is_block(modules):
```

Comment:
See if we can use utils from mmcv [here](https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/testing.py)